examples/hotrod/cmd/root.go

```diff
@@ -64,6 +65,7 @@ func init() {
 	RootCmd.PersistentFlags().StringVarP(&metricsBackend, "metrics", "m", "expvar", "Metrics backend (expvar|prometheus)")
 	RootCmd.PersistentFlags().DurationVarP(&fixDBConnDelay, "fix-db-query-delay", "D", 300*time.Millisecond, "Average lagency of MySQL DB query")
 	RootCmd.PersistentFlags().BoolVarP(&fixDBConnDisableMutex, "fix-disable-db-conn-mutex", "M", false, "Disables the mutex guarding db connection")
+	RootCmd.PersistentFlags().BoolVarP(&fixGetDriversConcurrently, "fix-get-driver-concurrent", "C", false, "Enable concurrency in driver::findNearest")
```

Comment:
I don't think it should be a boolean, rather an integer (default 1), exactly like `fixRouteWorkerPoolSize`

Reply:
SGTM. Will change.